includes/definitions/splat.yaml

```diff
@@ -0,0 +1,30 @@
+os: splat
```

Comment:
This should be secureplatform rather than splat (unless it's really called splat?). I've seen references to Gaia for the newer gen - what about that?

Reply:
Hi,
in network & security environments, checkpoint secureplatform is called splat (it's an acronism). I used this name becase it's how is known this device. But if you think is better change to secureplatform there is no problem.

I saw about CheckPoint Gaia, but for the moment I've not improved it. I saw that it has basic tests, but my company doesn't monitorize any gaia yet. I think in a mid future i will be able to improve it.
There are differences regarding to monitor these devices. Gaia contains OIDs which are not available in Gaia, for this reason I though to use a new os.